src/domain/home/moreInfo/MoreInfoLinkList.tsx

```diff
@@ -11,6 +11,7 @@ type MoreInfoLinkListProps = {
 const MoreInfoLinkList = ({ links }: MoreInfoLinkListProps) => {
   const { t, i18n } = useTranslation();
   const currentLanguage = getCurrentLanguage(i18n);
+  console.log({ currentLanguage });
```

Comment:
Do you want to keep this console.log here over the long term or was it just for debugging?

Reply:
Thanks, I will remove it 😄